### DISCUSSION TITLE
Next 13.3 how to pass data from client to server components

### DISCUSSION THREAD

**Discussion by Ravvsky** [vercel/next.js#49254] [Help]
### Summary

In page file I fetch some initial data and render some client components that are used for filtering. 

In a client component im generating some string that i need to add to api url to fetch filtered data. 
And I have no idea how i can get that data from client component on my page component (or any other server component)

**Comment by icyJoseph** [comment]
As of now, before server actions are taken into account, in Next.js 13.3 you'd need to something like:

- https://github.com/vercel/next.js/discussions/47227#discussioncomment-5342500

That is, using queries.

But in 13.4 there's server actions as alpha: https://nextjs.org/blog/next-13-4#server-actions-alpha

**Reply by AlfredMadere** [reply]
The first link is gold. How would you use server actions to get information from a client component to a server component (for example some state that the server component depends on to fetch data). It was my impression that server actions are just another way to fetch data from the backend.
